husky_extensions/ckeditor-extension.js

```diff
@@ -127,9 +128,30 @@
                         JustifyBlock: 'align-justify',
                         NumberedList: 'list-ol',
                         BulletedList: 'list',
-                        PasteFromWord: 'file-word-o',
+                        PasteFromWord: 'clipboard',
                         Styles: 'header',
                         Source: 'code'
+                    },
+                    relations = {
+                        Link: ['Unlink']
+                    },
+                    expandToolbar = function(toolbar) {
+                        for (var section in toolbar) {
+                            var index = -1;
+
+                            while (index++ < toolbar[section].length) {
+                                if (!relations[toolbar[section][index]]) {
+                                    continue;
+                                }
+
+                                toolbar[section].splice.apply(
+                                    toolbar[section],
+                                    [index + 1, 0].concat(relations[toolbar[section][index]])
+                                );
```

Comment:
This function call in general looks quite complicated to me. Isn't it possible to make that easier?